finalPrep/annotations.js

```diff
@@ -3,9 +3,19 @@
 
 //Build a Bear
 function buildABear(name, age, fur, clothes, specialPower) {
+  //declare a variable 'greeting' assign it to a string that uses interpolation to include the 'name'
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+//declare a variable 'demographics' that is an array with the values 'name' and 'age'
   var demographics = [name, age];
+//declare a variable 'powerSaying' assigning it to a string that uses concatenation to include the 'specialPower'
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+//declare a variable 'builtBear' assigned to an object with the following key/values
+  //basicInfo (string)
+  //clothes (string)
+  //exterior (string)
+  //cost (number)
```

Comment:
nice job taking this portion line by line. You may also think about including where the values come from- an argument, or something declared abouve?